<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LearnCPPOpenCVClass</class>
 <widget class="QMainWindow" name="LearnCPPOpenCVClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LearnCPPOpenCV</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QProgressBar {
    border: 1px solid #bbb;
    border-radius: 4px;
    min-height: 30px;
    text-align: center;
    padding: 2px;
    background-color: #f0f0f0;
}

QProgressBar::chunk {
    background-color: #4CAF50;
    width: 1px;
    margin: 0px;
    min-height: 30px;
}
QPushButton {
	font-size: 16px;
	min-height: 40px;
}
QLabel {
	font-size: 15px;
}
#label {
	font-size: 26px;
}
#lbOrigin, #lbRes {
	border: 1px dashed gray;
}
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,4,0,0,1,0">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>OpenCV Detection</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="lbOrigin">
          <property name="minimumSize">
           <size>
            <width>480</width>
            <height>480</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>480</width>
            <height>480</height>
           </size>
          </property>
          <property name="text">
           <string>原始图像</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbRes">
          <property name="minimumSize">
           <size>
            <width>480</width>
            <height>480</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>480</width>
            <height>480</height>
           </size>
          </property>
          <property name="text">
           <string>检测结果</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,1,1">
        <item>
         <widget class="QLabel" name="lbTarget">
          <property name="text">
           <string>目标类型：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbConf">
          <property name="text">
           <string>置信度：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbArea">
          <property name="text">
           <string>区域：</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QProgressBar" name="pgb">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>36</height>
           </size>
          </property>
          <property name="value">
           <number>100</number>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QTableWidget" name="table">
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>150</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>图像来源</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>目标类型</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>区域</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>置信度</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>时间</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>结果图像</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pbImages">
          <property name="text">
           <string>图片（目录）</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbVideo">
          <property name="text">
           <string>视频</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbCamera">
          <property name="text">
           <string>摄像头</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbStop">
          <property name="text">
           <string>停止</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="LearnCPPOpenCV.qrc"/>
 </resources>
 <connections/>
</ui>
